src/usecases/createHospital.js

```diff
@@ -0,0 +1,28 @@
+const createHospital = ({ getDb }) => async ({ name, trustId }) => {
+  const db = await getDb();
+
+  try {
+    console.log("Creating hospital for", name);
+    const createdHospital = await db.one(
+      `INSERT INTO hospitals
+          (id, name, trust_id)
```

Comment:
Do we need to specify the `id`?

Reply:
We use this pattern in other places as we pass in ```default```. We could refactor to not use id, but I think that would be a separate PR to refactor it everywhere to be consistent :)